lang/src/org/partiql/lang/eval/physical/PhysicalPlanCompilerImpl.kt

```diff
@@ -274,10 +275,12 @@ internal class PhysicalPlanCompilerImpl(
         val mapThunk = compileAstExpr(expr.exp)
         val bexprThunk: RelationThunkEnv = bexperConverter.convert(expr.query)
 
-        return thunkFactory.thunkEnv(expr.metas) { env ->
-            val relationTypeThunk = bexprThunk(env)
-            val relationType: RelationType = relationTypeThunk.relType
+        val relationType = when (expr.metas.containsKey(IsOrderedMeta.TAG)) {
```

Comment:
Could you please confirm that 
```
SELECT * FROM (SELECT a FROM <<{'a':1}>> as tbl ORDER BY tbl.a) as tbl2 
```
Returns a `BAG`. 

I wasn't too sure if the what the `isOrderedMeta` is going to be outside of the subquery.

Reply:
Yes, the query returns a bag. Here's the output:
```
<<
  {
    'a': 1
  }
>>
```
Here's another query showing that the ordering is confined in scope:

```sql
SELECT (SELECT a FROM <<{'a': 2}, {'a': 1}>> ORDER BY a) AS subQuery FROM [0]
```
returns:
```
<<
  {
    'subQuery': [
      {
        'a': 1
      },
      {
        'a': 2
      }
    ]
  }
>>
```

While `[0]` is sorted, the result is still a bag, while the projection item `subQuery` is a list because it uses ORDER BY.